.circleci/config.yml

```diff
@@ -34,6 +34,8 @@ jobs:
             ccache --version
             ccache --show-stats
             ccache --zero-stats
+            # cleanup cache (not clear!) - removes old cache
```

Comment:
Not really clear comment for me. What do you mean here?

Reply:
It is better to read ccache man